src/objectTranslations/npe03__Recurring_Donation__c-fr.objectTranslation

```diff
@@ -202,7 +202,7 @@
         <label>Mode de paiement</label>
         <name>PaymentMethod__c</name>
         <picklistValues>
-            <masterLabel>ACH/EFT</masterLabel>
+            <masterLabel>ACH</masterLabel>
             <translation>CCA/TEF</translation>
```

Comment:
I am assuming that "CCA/TEF" can be replaced with "CCA" similarly to the Spanish translation removal.

Reply:
Okay.  Pushed that change as well.